iSecretVault_webapp_ng2/src/app/app.component.spec.ts

```diff
@@ -1,16 +1,18 @@
 /* tslint:disable:no-unused-variable */
 ///<reference path="../../node_modules/@types/jasmine/index.d.ts"/>
 
-import { addProviders, async, inject } from '@angular/core/testing';
```

Comment:
These tests where removed because they were not working. 
Ultimately they should be fixed...